.github/workflows/run_performance_tests.yml

```diff
@@ -234,6 +234,15 @@ jobs:
           export KOSKI_VALPAS_OPPIJAOIDIT_FILENAME="valpas_qa_oppija_oidit.txt"
           mvn exec:java -Dexec.mainClass="fi.oph.koski.perftest.ValpasRandomPerusopetuksenOppijaGetter"
 
+      - name: Run perftest valpas-oppija-valintatulokset
```

Comment:
Tässä olisi mielestäni kaksi vaihtoehtoa
1) Luoda uusi perffitesti, joka ajetaan random_get_light kanssa ensimmäisenä testinä, jolloin jos valinnan tuloksia ei ole, voidaan perffitesti feilata jo tässä kohtaa. Tällöin riittää, että tarkistetaan esim. 10 tiedettyä oppijaa eikä 5000:ta.
2) Muutetaan olemassa olevien testejä niin, että validaatio on osa niitä. Tämän voi tehdä jos validointi ei vaikuta testiajon performanssiin ja kaikilla oppijoilla on valintatietoja.